data/data/openstack/topology/variables.tf

```diff
@@ -39,6 +39,10 @@ variable "masters_count" {
   type = string
 }
 
+variable "masters_schedulable" {
```

Comment:
I am thinking that this may be a useful variable to add to data/data/config.tf. It is obviously only be used by OpenStack now, but the concept is not specific to OpenStack.

Reply:
Duh. I don't know what I was looking at at the time. Sorry.